pkgs/applications/audio/cheesecutter/default.nix

```diff
@@ -8,13 +8,13 @@
 }:
 stdenv.mkDerivation rec {
   pname = "cheesecutter";
-  version = "unstable-2020-04-03";
+  version = "unstable-2021-02-27";
 
   src = fetchFromGitHub {
     owner = "theyamo";
     repo = "CheeseCutter";
-    rev = "68d6518f0e6249a2a5d122fc80201578337c1277";
-    sha256 = "0xspzjhc6cp3m0yd0mwxncg8n1wklizamxvidrnn21jgj3mnaq2q";
+    rev = "84450d3614b8fb2cabda87033baab7bedd5a5c98";
```

Comment:
lgtm, but does it have to be in this PR or could it be separate?

Reply:
Just realized these are D packages 👍

Reply:
What's the point in putting it in another PR? It's in separate commits anyway and since everything is related to dlang it makes sense to update everything in one go no?
If problems come up it could still be removed but as it is already done I would suggest to just roll with it.

Reply:
Hey, thank you guys, so glad to see this getting done :-)

I am pretty new to Nix* but would find it extremely difficult to part with Nix* (for my best config & environment experience) or D (for the little programming/dev that I do).

If it happens again I will try to report breakage sooner ... and may catch such things fairly early.
(I try do my personal development on nixpkgs unstable and want the latest of everything to catch breakage early, while developing I would like it to build/work on latest. Pinning / flakes etc. would only become relevant (as security) with a tag or release)

Thank you for your efforts.  @lionello thank you for off list help with dub2nix and do you think that perhaps for discoverability that could somehow get into nix packages? Sorry for being a bit off-topic, but this seems a great opportunity to ask :-) _[is there anything i need to do, like close something on github? i am assuming there isn't, i am new to that too and reluctantly but in an odd way gratefully drawn in to interact with it]_